clients/html/src/app/plan-filter/plan-filter.component.ts

```diff
@@ -130,7 +130,7 @@ export class PlanFilterComponent implements OnInit {
         this,
         this.employerDetails['sic']['standardIndustryCodeCode'],
         startDate,
-        'MA',
+        'ME',
```

Comment:
remove this as well, seems like we're not relying on this input in our api. Also, make sure to remove state from getPlansFor in plan provider service.

Reply:
have removed from plan-provider-service and plan-filter-component.